tools/accuracy_checker/configs/noise-suppression-denseunet-ll-0001.xml

```diff
@@ -0,0 +1 @@
+../../../models/intel/noise-suppression-denseunet-ll-0001/accuracy-check.yml
```

Comment:
@krodyush please rename soft link from configs/noise-suppression-denseunet-11-0001.xml to yml

Reply:
Thanks!